stac_generator/base/__init__.py

```diff
@@ -0,0 +1,10 @@
+from stac_generator.base.generator import CollectionGenerator, ItemGenerator, StacSerialiser
+from stac_generator.base.schema import SourceConfig, StacCollectionConfig
+
+__all__ = (
```

Comment:
It's for convenience only. You can run 

```python
from stac_generator.base import CollectionGenerator
```
instead of 

```python
from stac_generator.base.generator import CollectionGenerator 
```
if you export the names